tests/vmi_configuration_test.go

```diff
@@ -106,11 +106,13 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 	Context("with all devices on the root PCI bus", func() {
 		It("[test_id:4623]should start run the guest as usual", func() {
-			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-			vmi.Annotations = map[string]string{
-				v1.PlacePCIDevicesOnRootComplex: "true",
-			}
-			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
+			vmi := libvmi.NewCirros(
+				libvmi.WithNamespace(util.NamespaceTestDefault),
```

Comment:
It should work without this, if you follow `tests.RunVMIAndExpectLaunch` you will see it is specified on creation there, unrelated to what is set in the vmi spec.